api/subscriptions/permissions.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import permissions
+
+from osf.models.notifications import NotificationSubscription
+
+
+class IsOwner(permissions.BasePermission):
+
+    def has_object_permission(self, request, view, obj):
+        assert isinstance(obj, NotificationSubscription), 'obj must be an NotificationSubscription; got {}'.format(obj)
```

Comment:
'obj must be an NotificationSubscription; got {}' -> 'obj must be **a** NotificationSubscription; got {}'